<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-A6GIKER" ignored="8" name="Test" tests="8" failures="6" timestamp="2022-11-30T02:17:42 IST" time="53.168" errors="0">
  <testcase name="UserHomeTest" time="14.345" classname="testcases.UserHomePageTest"/>
  <testcase name="ManageProductTest" time="0.001" classname="testcases.ManageProductPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.ManageProductPageTest.ManageProductTest(ManageProductPageTest.java:27)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ManageProductTest -->
  <testcase name="ViewProductTest" time="0.001" classname="testcases.ViewProductPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.ViewProductPageTest.ViewProductTest(ViewProductPageTest.java:27)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ViewProductTest -->
  <testcase name="AdminHomeTest" time="0.0" classname="testcases.AdminHomePageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.AdminHomePageTest.AdminHomeTest(AdminHomePageTest.java:26)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AdminHomeTest -->
  <testcase name="LoginSuccessTest" time="0.001" classname="testcases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.LoginPageTest.LoginSuccessTest(LoginPageTest.java:25)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginSuccessTest -->
  <testcase name="LoginValidationTest" time="0.001" classname="testcases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.LoginPageTest.LoginValidationTest(LoginPageTest.java:35)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginValidationTest -->
  <testcase name="LoginValidationTest1" time="0.0" classname="testcases.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testcases.LoginPageTest.LoginValidationTest1(LoginPageTest.java:45)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginValidationTest1 -->
  <testcase name="LoginValidationTest2" time="1.228" classname="testcases.LoginPageTest"/>
  <testcase name="@BeforeTest DataSetUP" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest DataSetUP -->
  <testcase name="@BeforeTest DataSetUP" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest DataSetUP -->
  <testcase name="@BeforeTest DataSetUP" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest DataSetUP -->
  <testcase name="@AfterTest DataTearDown" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterTest DataTearDown -->
  <testcase name="@AfterTest DataTearDown" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterTest DataTearDown -->
  <testcase name="@AfterTest DataTearDown" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterTest DataTearDown -->
  <testcase name="@BeforeTest DataSetUP" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest DataSetUP -->
  <testcase name="@AfterTest DataTearDown" time="0.0" classname="testcases.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterTest DataTearDown -->
</testsuite> <!-- Test -->
